src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/security/CaseloadAccess.kt

```diff
@@ -1,27 +1,14 @@
 package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.security
 
-import org.springframework.web.context.request.RequestContextHolder
-import org.springframework.web.context.request.ServletRequestAttributes
-import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.config.getBvlsRequestContext
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.User
 import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.UserType
 
 /**
  * Caseload access checks are only relevant for prison users. If the current user is not a prison user then it will just
  * be ignored.
  */
-fun checkCaseLoadAccess(prisonCode: String) {
```

Comment:
The cancel scenario showed a floor with this approach.  There won't always be a user in the request attributes e.g. service User.  However we can use the User passed into the service calls directly as they will always be available to be passed through.